### PR TITLE
Fix settings for sattools_repo and cdn

### PR DESCRIPTION
**Pull request by swadeley**
Hello

fixing settings path to `conf/repos` and `conf/robotelo` in hosts.py

### CODE REVIEW

**Review by jameerpathan111 — APPROVED** [review]
Ack

File: robottelo/hosts.py [hunk 1]
```diff
@@ -647,8 +647,8 @@ def virt_who_hypervisor_config(
                 'repository-set': constants.REPOSET['rhst7'],
                 'repository': constants.REPOS['rhst7']['name'],
                 'repository-id': constants.REPOS['rhst7']['id'],
-                'url': settings.sattools_repo['rhel7'],
-                'cdn': bool(settings.cdn or not settings.sattools_repo['rhel7']),
+                'url': settings.repos.sattools_repo['rhel7'],
+                'cdn': bool(settings.robottelo.cdn or not settings.repos.sattools_repo['rhel7']),
```
Comment (devendra104) [inline robottelo/hosts.py:651]: Can't we do this without type conversion?

```
>>> settings.robottelo.cdn or not settings.repos.sattools_repo['rhel7']
True
>>> settings.robottelo.cdn
True
>>> type(settings.robottelo.cdn)
<class 'bool'>
```
```suggestion
                'cdn': settings.robottelo.cdn or not settings.repos.sattools_repo['rhel7'],
```

Comment (swadeley) [reply robottelo/hosts.py:651]: I think that line comes from this git commit:

`commit 5bd6d20b2ff84c5035c09286b4c8d2e8933f9587
Author: Mike Shriver <mshriver@redhat.com>
Date:   Wed Jun 2 08:01:25 2021 -0400

    Removing VirtualMachine class and methods
    
    Move robottelo.cli.factory.setup_virtual_machine into ContentHost class`

but that line exists in other files before that, so I think it was just moved.


I will wait for @JacobCallahan to comment.

Comment (JacobCallahan) [reply robottelo/hosts.py:651]: Correct on both accounts. It was moved and can be simplified!

Comment (swadeley) [reply robottelo/hosts.py]: OK, changes made, thank you

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**